src/graphics/program-lib/programs/standard.js

```diff
@@ -637,9 +637,10 @@ var standard = {
                 attributes["vertex_texCoord" + i] = "TEXCOORD" + i;
                 code += chunks["uv" + i + "VS"];
                 codeBody += "   vec2 uv" + i + " = getUv" + i + "();\n";
-            }
-            if (useUnmodifiedUv[i]) {
-                codeBody += "   vUv" + i + " = uv" + i + ";\n";
+
+                if (useUnmodifiedUv[i] === true || useUnmodifiedUv[i] === undefined) {
```

Comment:
feels bit strange to do it if it's set to undefined. Should this maybe be fixed elsewhere in the code and make this to be set to true?

Reply:
I'll move it to the place where forceUv1 is applied (just above)